.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!--
+Thanks for contributing!
+-->
+
+Some things to make sure you've done:
+* Ran `swiftlint` and ensured that your additions/deletions are not causing any
+new warnings or errors.
+* You did _not_ include any changes to the generated documentation (we update
+the documentation as part of a separate commit).
+* If you added new functionality, then you've added new tests that correctly
+verify its correctness.
+* You ran `make test` and ensured that the driver still passes all of its
+pre-existing tests.
+* You read `CONTRIBUTING.md`!
```

Comment:
I would suggest bumping this to the top.

Reply:
Done.